### ISSUE TITLE
If a user is trying to donate to a verified project with a not-whitelisted GIVbacks token, they get a popup explaining that they might not get GIVbacks

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#606]
The user selects the (non-whitelisted) to token and clicks "donate". They get a popup that says:

"The token you have selected to donate is not eligible for GIVbacks. To see the full list of eligible tokens, please refer to our GIVbacks documentation."

Where "GIVbacks documentation" is a link to https://docs.giveth.io/giveconomy/givbacks

And there is a button that says "DONATE ANYWAY" that prompts the tx
And smaller text under that says "CANCEL" that closes the modal and lets the user pick another token.

**Comment by MoeNick** [comment]
Is that decision was already made from the forum post? 

- Do we have to have a token white list  in the backend? 

- Should we only have one white list fir every project? Or it should be change for the type of project ? 

- How are we going to update the white list? Once a day? From which source? 

@divine-comedian @mohammadranjbarz you may make it more clear

**Comment by mohammadranjbarz** [comment]
> Is that decision was already made from the forum post?
> 
> * Do we have to have a token white list  in the backend?
> * Should we only have one white list fir every project? Or it should be change for the type of project ?
> * How are we going to update the white list? Once a day? From which source?
> 
> @divine-comedian @mohammadranjbarz you may make it more clear

1. Not as you think, we should make it and implement a query to frontend use it
2. I think we should have one whitelist for verified tokens and it's not related to projects, whitelist for every project just makes complexity, and with the things that talked in forum post, I can,t find any reason for that 
3. @laurenluz  or  @divine-comedian  Can answer that

**Comment by laurenluz** [comment]
The details of the whitelist haven't been confirmed yet on the forum, but we will have them for sure!!

This issue is blocked by completion of the GIVbacks token whitelist

**Comment by laurenluz** [comment]
ok! @MoeNick, as per my proposal in the forum: https://forum.giveth.io/t/which-tokens-donated-are-eligible-for-givbacks/231/9 (which seems very likely to pass)

We're calling it the "GIVbacks Token List" and it will be same for every verified project. The list will include the tokens we have already added to the DApp (in the donation drop down menu) and will list them in a post on the forum (I will create it). When people want to add new tokens, they can comment in the forum and we will vote to determine if it should be added.

We can make a the frequency by which we update the list based on how frequently we get requests to add new tokens in the forum.

**Comment by mateodaza** [comment]
I'm trying to come up with a great UX for this but struggling to make it the less annoying as possible, do you have any suggestion?

I was thinking instead of a popup, it's just a clearer message near the button

**Comment by laurenluz** [comment]
What if we made it a little red text alert thing under here that says "This Token is not eligible for GIVbacks. Learn More." And "Learn More" is clickable and it opens the popup I described in my first issue.

![image](https://user-images.githubusercontent.com/75490971/148003679-4868dad3-d33e-4a43-89f4-b19181cfe7eb.png)

**Comment by laurenluz** [comment]
FYI - the token list is up in the forum! https://forum.giveth.io/t/givbacks-token-list/253

I edited the suggestion in the issue so that the list is here, rather than in the documention.

**Comment by laurenluz** [comment]
@markoprljic FYI

**Comment by markoprljic** [comment]
Changes made to accommodate the scenario of donating with GIVbacks eligible and not eligible tokens.

1. User should be able to see this in the token list dropdown 
![Image 2022-01-12 at 2 03 16 pm](https://user-images.githubusercontent.com/5704285/149145795-b544265c-da78-42c9-8d45-9e00f2bbf1cc.jpg)

2. When **not eligible** token selected
![Image 2022-01-12 at 2 04 07 pm](https://user-images.githubusercontent.com/5704285/149145904-2f622559-de4b-4e32-9d3c-cf06419159a2.jpg)

3. When **eligible** token selected
![Image 2022-01-12 at 2 04 15 pm](https://user-images.githubusercontent.com/5704285/149145920-b0b44e29-c3d5-4a00-a0a5-ed8dc4c91dae.jpg)

Learn more opens a modal with more info. Please use standard modal component and add copy about What are GIVbacks to inform user about it, just briefly. DO NOT include any other links that will lead away from the donation process user is currently in.

@laurenluz @MoeNick 


P.S. I just realised this issue is for old dapp :/ Nevermind, it applies to new version as well :)

**Comment by laurenluz** [comment]
@markoprljic the GIVbacks token list is the same as our drop-down list of tokens (right? @mateodaza @mohammadranjbarz) so the tokens that won't be eligible are ones that user copy/pastes in the address for. I don't think we need to have the "all" and "GIVbacks eligible" selectors.

I do think we should keep the message "This Token is/is not eligible for GIVbacks Learn more"

I also totally understand that we don't want to send the user away from the donation page... but we should someone make it easy/possible for the to navigate from a page referring to the token list... to [the token list itself](https://forum.giveth.io/t/givbacks-token-list/253)... then if they want to request a token to be added to the list, they can do so.

I can write copy for the modal, but waiting on your feedback.

**Comment by MoeNick** [comment]
We'd better continue it on TP version, so I move the @laurenluz comment to a new issue and transfer it there, it's in our current sprint.
